src/log.js

```diff
@@ -597,7 +597,11 @@ export function user(opt_element) {
  */
 function getUserLogger(suffix) {
   if (!logConstructor) {
-    throw new Error('failed to call initLogConstructor');
+    const errorMsg = 'failed to call initLogConstructor.' +
+          (getMode().test || getMode().localDev) ?
+      ' Run "gulp clean && gulp css && gulp build" and try again' :
+      '';
```

Comment:
Unfortunately, I don't think this is the right fix, since it is likely papering over a real bug somewhere else, where `initLogConstructor` isn't being called even though it should. In addition, calling `gulp css` is redundant` since `gulp build` also builds all CSS files. Finally, it's unnecessary to build the runtime in order to run unit tests, so I'm afraid this isn't the right fix.

Can you send me the exact repro steps for this via a new or existing GH issue? I can look into what might be causing the failure.

Reply:
I don't have repro steps at this time, but I do know that i see this error very frequently, as does the rest of my team. I always just run `gulp clean && gulp css` and that fixes the issue, although in some rare cases I think `gulp build` might have been needed too, but that may have been a long time ago. I think just doing `gulp clean && gulp css` is probably sufficient

Reply:
The problem most often occurs when switching between feature branches in my experience, but point taken. I'll close.

Reply:
When you switch between feature branches, there are no guarantees that your old build will continue to work as expected, since things like version numbers, etc. are likely to have changed.